FirstWeek/PalindromeAndCaesarCipher.java

```diff
@@ -0,0 +1,212 @@
+/*
+MICP Summber 2017
+Author: Xinyi(Cindy) Chen
+Email: xinyi.chencareer@gmail.com
+Language: Java
+*/
+
+/**********WEEK ONE***********/
+/*
+1.1 A Palindrome is a number that reads the same forward as it does in reverse. For example, 12321 is a palindrome because it reads the same going from left to right as it does going right to left. Most numbers can be made into palindromes by taking the original number, reversing the digits, then adding the original number to the reversed digits. If that number isn’t a palindrome, then repeat the same process until you reach a palindrome. Most numbers can be made into palindromes by repeating this process, but some numbers don’t converge. During this process, if the number becomes larger than 1,000,000 and still doesn’t form a palindrome, then return false.
+*/
+/*
+1. Examples
+normal: 123 -> 123 + 321 = 444 ture
+single number: 9 -> 9 + 9 = 18; 18 + 81 = 99 true
+negative number: -10 not palindrome false
+large number: 999,999 -> 999,999 + 999,999 exceed 1,000,000, false
```

Comment:
999999 is a palindrome though, so you wouldn't need to reverse it and it's considered a valid input/success case